public/css/confirm.css

```diff
@@ -1,29 +1,39 @@
 @import url(https://fonts.googleapis.com/css?family=Roboto+Condensed:400&subset=latin, latin-ext);
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;700&display=swap");
```

Comment:
You don't need these CSS imports if you have the `<link>` tags in the HTML. I'd advise you stick with the `<link>`s because that's what the rest of our pages use